CMakeLists.txt

```diff
@@ -21,66 +24,55 @@ IF (CMAKE_CXX_COMPILER_ID STREQUAL "GNU"
     ADD_DEFINITIONS (-Wall)
 ENDIF ()
 
-# Fixes a lot of annoying warnings about auto_ptr deprecation
-ADD_DEFINITIONS (-DBOOST_NO_AUTO_PTR)
-
-SET (BOOST_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+SET (BOOST_DIR ${CMAKE_CURRENT_SOURCE_DIR}/boost)
 SET (MYSQL_SRC ${CMAKE_CURRENT_SOURCE_DIR}/mysql)
 SET (MYSQL_BIN ${CMAKE_BINARY_DIR}/mysql)
 FILE (MAKE_DIRECTORY ${MYSQL_BIN})
 
-INCLUDE_DIRECTORIES (
-    BEFORE SYSTEM
-    ${MYSQL_BIN}/include
-    ${MYSQL_SRC}/include
-    ${MYSQL_SRC}/libbinlogevents/export
-    ${BOOST_DIR}/boost_1_69_0/boost
-)
-ADD_CUSTOM_TARGET (
-    fix_static_build
-    WORKING_DIRECTORY ${MYSQL_BIN}
-    COMMAND sed -i '/GET_TARGET_PROPERTY.*libmysql/s/libmysql/perconaserverclient/' ${MYSQL_SRC}/scripts/CMakeLists.txt
-)
 ADD_CUSTOM_TARGET (
-    mysql_configure
-    WORKING_DIRECTORY ${MYSQL_BIN}
-    DEPENDS fix_static_build
-    COMMAND "${CMAKE_COMMAND}"
+        mysql_configure
+        WORKING_DIRECTORY ${MYSQL_BIN}
+        COMMAND "${CMAKE_COMMAND}"
         -DCMAKE_BUILD_TYPE=Release
         -DDISABLE_SHARED=1
         -DENABLED_PROFILING=0
         -DWITHOUT_SERVER=1
         -DWITH_CLIENT_PROTOCOL_TRACING=0
         -DWITH_DEFAULT_FEATURE_SET=0
-        -DWITH_SSL=bundled
+        -DWITH_SSL=system
         -DWITH_ZLIB=bundled
         -DWITH_LIBEVENT=bundled
         -DWITH_LZ4=bundled
         -DWITH_PROTOBUF=bundled
         -DWITH_EDITLINE=bundled
-        -DDOWNLOAD_BOOST=1 -DWITH_BOOST=${BOOST_DIR}
+        -DDOWNLOAD_BOOST=1
+        -DDOWNLOAD_BOOST_TIMEOUT=1200
+        -DWITH_BOOST=${BOOST_DIR}
         ${MYSQL_SRC}
 )
+
 ADD_CUSTOM_TARGET (
-    mysqlclient_build
-    WORKING_DIRECTORY ${MYSQL_BIN}
-    DEPENDS mysql_configure
-    COMMAND ${CMAKE_COMMAND} --build . --target perconaserverclient
+        binlogevents_build
+        WORKING_DIRECTORY ${MYSQL_BIN}
+        DEPENDS mysql_configure
+        COMMAND ${CMAKE_COMMAND} --build . --target binlogevents_static
 )
+
 ADD_CUSTOM_TARGET (
-    binlogevents_build
-    WORKING_DIRECTORY ${MYSQL_BIN}
-    DEPENDS mysql_configure
-    COMMAND ${CMAKE_COMMAND} --build . --target binlogevents_static
+        mysql_client_build
+        WORKING_DIRECTORY ${MYSQL_BIN}
+        DEPENDS binlogevents_build
+        COMMAND ${CMAKE_COMMAND} --build . --target perconaserverclient
```

Comment:
It seems it's a formatting change and it is not related to Percona 8 support. Please remove it.